griptape/artifacts/info_artifact.py

```diff
@@ -7,7 +7,15 @@
 
 @define
 class InfoArtifact(BaseArtifact):
```

Comment:
Is this not a TextArtifact because we don't want it to match `isinstance` on `TextArtifact` in order to make code branching a little easier?

Reply:
Correct. If we wanted to re-use code we could create a `BaseTextArtifact` but I'd like to keep things flat and it feels unnecessary at this time.